contracts/Liquidator.sol

```diff
@@ -1,41 +1,82 @@
-pragma solidity ^0.5.16;
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.10;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
-import "./ComptrollerInterface.sol";
-import "./VAIControllerInterface.sol";
-import "./VBNB.sol";
-import "./VBep20.sol";
 import "./Utils/ReentrancyGuard.sol";
 import "./Utils/WithAdmin.sol";
-import "./Utils/SafeMath.sol";
-import "./Utils/IBEP20.sol";
-import "./Utils/SafeBEP20.sol";
+
+interface IComptroller {
```

Comment:
Why not use the `IComptroller` interface in `contracts/ComptrollerInterface`?

Reply:
because it returns `unit` instead of unit256?

Reply:
uint is uint256 under the hood. The reason might be because he upgraded to latest version and cannot import any contracts/interfaces with different one

Reply:
that makes sense

Reply:
@coreyar can I treat this thread as resolved?